lib/src/connector/course_connector.dart

```diff
@@ -101,38 +101,57 @@ class CourseConnector {
       //取得學期資料
       nodes = courseNodes[0].getElementsByTagName("td");
       SemesterJson semester = SemesterJson();
-      semester.year = nodes[1].text;
-      semester.semester = nodes[2].text;
+
+      // the title String of the first course table was stored seperately in its <td> element,
```

Comment:
```suggestion
      // Previously, the title string of the first course table was stored separately in its `<td>` element,
```

Adding "Previously" to the comment makes it easier to understand. Also fixed capitalization and spelling issues.